guilford_court_commitment.yaml

```diff
@@ -0,0 +1,408 @@
+        type: "ol.durationdays"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "TOTAL LENGTH OF SUPERVISION"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "TOTAL LENGTH OF SUPERVISION"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      ol.type:
+        type: "ol.type"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "TYPE OF OLD PE DATE CODE"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "TYPE OF OLD PE DATE CODE"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      ol.newperiod:
+        type: "ol.newperiod"
+        column: "NEW PERIOD OF SUPERVISION FLAG"
+    name: "supervisions"
+
+  locations:
+    fqn: "ol.location"
+    entitySetName: "GuilfordLocations"
+    propertyDefinitions:
+      general.id: # most serious offense code
+        type: "general.id"
+        column: "CO OF CONV MOST SERIOUS OFFNSE"
+      ol.county: # county of most serious offense
+        type: "ol.county"
+        column: "CO OF CONV MOST SERIOUS OFFNSE"
+    conditions:
+    - !<conditions.BooleanRegexCondition>
+      column: "CO OF CONV MOST SERIOUS OFFNSE"
+      pattern: "\\?+"
+      reverse: true
+    name: "locations"
+
+
+
+associationDefinitions:
+ 
+
+
+  subjectof:
+    fqn: "ol.subjectof"
+    entitySetName: "GuilfordSubjectOf"
+    src: "offenders"
+    dst: "jailstays"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.HashTransform>
+          columns: ["COMMITTED FIRST NAME", "COMMITTED MIDDLE NAME", "COMMITTED LAST NAME", "COMMITTED NAME SUFFIX", "OFFENDER NC DOC ID NUMBER", "OFFENDER ADMISSION/INTAKE DATE"]
+          hashFunction: "sha256"
+    name: "subjectof"
+
+  sentencedwith:
+    fqn: "ol.sentencedwith"
+    entitySetName: "GuilfordSentencedWith"
+    src: "offenders"
+    dst: "sentences"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.HashTransform>
+          columns: ["COMMITTED FIRST NAME", "COMMITTED MIDDLE NAME", "COMMITTED LAST NAME", "COMMITTED NAME SUFFIX", "OFFENDER NC DOC ID NUMBER", "OFFENDER NC DOC ID NUMBER"]
+          hashFunction: "sha256"
+    name: "sentencedwith"
+
+  partof:
+    fqn: "ol.partof"
+    entitySetName: "GuilfordPartOf"
+    src: "sentencecomponents"
+    dst: "sentences"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.HashTransform>
+          columns: ["COMMITMENT PREFIX", "OFFENDER NC DOC ID NUMBER"]
+          hashFunction: "sha256"
+    name: "partof"
+
+  partof2:
+    fqn: "ol.partof"
+    entitySetName: "GuilfordPartOf"
+    src: "sentencecomponents"
+    dst: "jailstays"
```

Comment:
shouldn't this be sentence part of jailstay?

Reply:
sentence components are partof sentences and sentencecomponents are partof jailstays. Kim had it this way in the wiki, and when I asked she suggested to keep it this way. If you think that sentences being partof jailstays makes more sense, I am definitely open to learning from that discussion